keps/sig-network/2086-service-internal-traffic-policy/README.md

```diff
@@ -167,6 +167,10 @@ Beta:
 * feature gate `ServiceInternalTrafficPolicy` is enabled by default.
 * consensus on how internalTrafficPolicy overlaps with topology-aware routing.
 
+GA:
```

Comment:
Adding GA criteria now so we know what to work towards in v1.24 but keeping GA milestone in kep.yaml to v1.25